config/dialog/dialog_blacksmith.yml

```diff
@@ -51,49 +51,52 @@ dialogs:
         - quest intro gear
         - flag innkeeper_dislike > 0
         script:
-        - '0 out blacksmith_hello  Hello there, what can i do for you?'
+        - 0 out Hello there, what can i do for you?
         - 5 option reason_dislike
     gear:
         requirements: 
         - quest intro gear
         script:
-        - '0 out blacksmith_hello  Hello there, what can i do for you?'
+        - 0 out Hello there, what can i do for you?
         - 5 option reason
     response_innkeeper_dislike:
         script:
-        - '0 me The innkeeper sent me.'
-        - '20 out blacksmith_innkeeper And I think that means that you are looking for some gear?'
+        - 0 me The innkeeper sent me.
+        - 20 out And I think that means that you are looking for some gear?
         - 0 option looking_dislike   
     response_innkeeper:
         script:
-        - '0 me The innkeeper sent me.'
-        - '20 out blacksmith_innkeeper And I think that means that you are looking for some gear?'
+        - 0 me The innkeeper sent me.
+        - 20 out And I think that means that you are looking for some gear?
         - 0 option looking   
     response_sarcastic_dislike:
         script:
-        - '0 me what else?'
-        - '20 out response_sarcastic_dislike I can see why the Innkeeper doesn''t like you, but I have some gear left over for people such as yourself...'
+        - 0 me what else?
+        - 20 out I can see why the Innkeeper doesn''t like you, but I have some gear left over for people such as yourself...
         - 0 zap dislike_gear
     response_sarcastic:
         script:
-        - '0 me what else?'
-        - '20 out response_sarcastic Hmmm, I would appreciate it if you could be a little less sarcastic, but I have some gear left over for people such as yourself...'
+        - 0 me what else?
+        - 20 out Hmmm, I would appreciate it if you could be a little less sarcastic, but I have some gear left over for people such as yourself...
         - 0 zap dislike_gear
     dislike_gear:
         script:
-        - '20 out dislike_gear I found a crappy bow and a rusty sword, you can chose which piece of trash you want.'
+        - 20 out I found a crappy bow and a rusty sword, you can chose which piece of trash you want.
```

Comment:
I think "crappy" is not appropriate here since its english slang. Perhaps something like:
`I found this old bow and rusty sword. You may choose which piece of trash you like"